inc/customizer/class-storefront-customizer.php

```diff
@@ -786,11 +786,12 @@ public function get_css() {
 				color: ' . $storefront_theme_mods['footer_text_color'] . ';
 			}
 
-			.site-footer a:not(.button) {
+			.site-footer a:not(.button),
+			.site-footer a:not(.button):not(.components-button) {
 				color: ' . $storefront_theme_mods['footer_link_color'] . ';
 			}
 
-			.site-footer h1, .site-footer h2, .site-footer h3, .site-footer h4, .site-footer h5, .site-footer h6 {
+			.site-footer h1, .site-footer h2, .site-footer h3, .site-footer h4, .site-footer h5, .site-footer h6, .widget .widget-title, .widget .widgettitle {
```

Comment:
These two new classes should be prepended with the `.site-footer` selector, no?
```suggestion
			.site-footer h1, .site-footer h2, .site-footer h3, .site-footer h4, .site-footer h5, .site-footer h6, .site-footer .widget .widget-title, .site-footer .widget .widgettitle {
```
Otherwise it's being applied to sidebar widgets as well:
![imatge](https://user-images.githubusercontent.com/3616980/74917645-1ccd4280-53c8-11ea-90e7-975341ea0636.png)